.github/workflows/integration.yml

```diff
@@ -33,6 +33,49 @@ jobs:
         name: astar-${{ matrix.os }}-x86_64
         path: target/release/astar-collator
 
+  runtime-benchmarks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hecrj/setup-rust-action@v1
+      with:
+        rust-version: 'nightly-2021-06-29'
```

Comment:
The rust version is something that we will change relatively often and the string value is used multiple times, what about reading it as a variable? Perhaps as a matrix or env var.
https://docs.github.com/en/actions/learn-github-actions/contexts

Reply:
You're right. I thought about it as I added this but was too lazy in the moment.

In my defense, I did plan to look into this after I see that it worked :laughing: